bucket/haskell-language-server.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "2.7.0.0",
+    "description": "The official language server for Haskell.",
+    "homepage": "https://github.com/haskell/haskell-language-server",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/haskell/haskell-language-server/releases/download/2.7.0.0/haskell-language-server-2.7.0.0-x86_64-mingw64.zip",
+            "hash": "34a54059bf1de223c1fd17e0d026ce87942393b97e23dabc2bed78f3000e4a0a"
+        }
+    },
+    "bin": "haskell-language-server-wrapper.exe",
```

Comment:
```suggestion
```

Pretty sure there's no need to add this shim if the entire folder is added to PATH already!

Reply:
Of course! 😄